index.html

```diff
@@ -16,17 +16,21 @@
 <body>
     <header>
         <ul>
-            <li><img src="pics/icon/facebook.png" alt="Facebook"></li>
-            <li><img src="pics/icon/twitter.png" alt="Twitter"></li>
-            <li>English</li>
-            <li>My page</li>
-            <li>Logout</li>
+            <li><a href="#"><img id="face" src="pics/icon/facebook.png" alt="Facebook"></a></li>
+            <li><a href="#"><img id="twi" src="pics/icon/twitter.png" alt="Twitter"></a></li>
+            <li><a href="#">English</li></a>
+            <li><a href="#">My page</li></a>
+            <li><a href="#">Logout</li></a>
         </ul>
     </header>
 
     <div class="menuNav">
-        <nav id="navBar" class="navBar">     
-            <a href="about.html" class="about">About</a>
+
+        <button class="campaign">CC Campaign</button>
```

Comment:
- [optional] nice work. please, for you to really align with the main design, the speaker section of the mobile version should be displaying two speakers. You can add the more button functionality if you want to show extra speakers. Nice job.